.github/workflows/ci.yml

```diff
@@ -43,6 +43,35 @@ jobs:
       - name: Run tests
         run: sh ci/run.sh ${{ matrix.os }}
 
+  create_issue:
+    runs-on: ubuntu-latest
+    # the combination of these two lines seems to do the "if build_and_test failed, run this one"
+    needs: build_and_test
+    if: always() && github.event_name == 'schedule' && needs.build_and_test.result == 'failure'
+
+    steps:
+      - name: Get today's date
+        id: date
+        run: |
+          echo "::set-output name=today::$(date "+%Y-%m-%d")"
+
+      - name: Create an issue
+
+        uses: actions-ecosystem/action-create-issue@v1
```

Comment:
Could you avoid using third-party actions (perhaps using GitHub CLI)? There's a security concern around it.

Reply:
ah, yep, makes sense (also TIL).